<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmShadowClock</class>
 <widget class="QWidget" name="frmShadowClock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FrmPieTImeWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="ShadowClock" name="shadowClock1" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="ShadowClock" name="shadowClock2" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="ShadowClock" name="shadowClock3" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="ShadowClock" name="shadowClock4" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="ShadowClock" name="shadowClock5" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="ShadowClock" name="shadowClock6" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ShadowClock</class>
   <extends>QWidget</extends>
   <header>shadowclock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
